src/components/WebexActivity/WebexActivity.stories.js

```diff
@@ -90,3 +90,8 @@ export const MultiLine = Template.bind({});
 MultiLine.args = {
   activityID: 'activity3',
 };
+
+export const AdaptiveCardActivity = Template.bind({});
```

Comment:
all of them are activities, as we are not using MultiLineActivity, let's not use AdaptiveCardActivity, instead, use just AdaptiveCard